en/docs/WIP/create-and-publish-awslambda-api.md

```diff
@@ -0,0 +1,122 @@
+   </div> 
+</html>
+
+### Step 1 - Design a REST API
+
+1. Sign in to the API Publisher Portal.
+   
+    `https://<hostname>:9443/publisher` 
+   
+    Example: `https://localhost:9443/publisher`
+
+    Use your username and password to sign in.
+
+2. Click **CREATE API** and then click **Design a New REST API**.
+
+    [![Create a new REST API]({{base_path}}/assets/img/Learn/create-api-design-rest-api-link.png)]({{base_path}}/assets/img/Learn/create-a-rest-api.jpg)
+
+3. Enter API details and click **CREATE**.  
+
+    [![Create a test API]({{base_path}}/assets/img/Learn/create-test-api.png)]({{base_path}}/assets/img/Learn/create-test-api.png)
+
+    <html>
+      <div class="admonition note">
+         <p class="admonition-title">Note</p>
+         <p>You don't need to enter the Endpoint at this moment.</p>
+      </div> 
+    </html>
+
+Now, you have created a new REST API successfully. 
+
+### Step 2 - Add AWS Lambda endpoint
+
+1. Goto **Endpoints**
+2. Add **AWS Lambda** endpoint type
+
+    [![Select AWS Lambda endpoint]({{base_path}}/assets/img/Learn/Tutorials/endpoint-select-awslambda-endpoint.png)]({{base_path}}/assets/img/Learn/Tutorials/endpoint-select-awslambda-endpoint.png)
+
+3. Select **Access Method**
+
+    AWS SDK needs AWS credentials to invoke AWS Lambda functions. Access Method defines how you provide those AWS credentials. You can provide AWS credentials manually by selecting **Using stored AWS credentials** method. But if WSO2 API Manager is running on an Amazon EC2 instance, you can select **Using IAM role-supplied temporary AWS credentials** method.
+
+    <html>
+      <div class="admonition note">
+         <p class="admonition-title">Note</p>
+         <p>To use IAM role-supplied temporary AWS credentials method, you should attach an IAM role to grant permissions to applications running on Amazon EC2 instance.</p>
+         <p>For more information on attaching IAM role to EC2, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html">Using an IAM Role to Grant Permissions to Applications Running on Amazon EC2 Instances</a>.</p>
+      </div> 
+    </html>
+
+4. Click **Save**
+
+    [![Save AWS Lambda endpoint]({{base_path}}/assets/img/Learn/Tutorials/endpoint-awslambda-save.png)]({{base_path}}/assets/img/Learn/Tutorials/endpoint-awslambda-save.png)
+
+    <html>
+      <div class="admonition note">
+         <p class="admonition-title">Note</p>
+         <p>You'll get an error message if you have not set the Access Method properly.</p>
+      </div> 
+    </html>
+
+### Step 3 - Map Function-ARNs to Resources
+
+<html>
+   <div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>
+         For more information on ARNs, see 
+         <a href="https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html">Amazon Resource Names (ARNs)</a>
+         .
+      </p>
+   </div> 
+</html>
+
+1. Goto **Resources**.
+2. Configure resources.
```

Comment:
Follow steps below to configure resources.

Reply:
changed